packages/arb-token-bridge-ui/src/tokenLists.tsx

```diff
@@ -123,29 +115,35 @@ export async function fetchTokenListFromURL(
 
     if (!validateTokenList(data)) {
       console.warn('Token List Invalid', data)
-      return { isValid: false, data }
+      return { isValid: false, data, tokenListURL }
```

Comment:
NIT: I know it wasn't introduced in this PR, but do we ever need to access `tokenListURL` if it's not valid? If not, I guess we can simply use `if (tokenListURL)` wherever we use the hook

Reply:
Yeah, we don't need to access `tokenListURL` if the list is invalid. It was just added coz of typescript signature. I've marked the `tokenListUrl` as optional now and updated this invalid schema return to the original.